src/main/java/io/fusionauth/samlv2/service/DefaultSAMLv2Service.java

```diff
@@ -1006,7 +1334,6 @@ private String signXML(PrivateKey privateKey, X509Certificate certificate, Algor
     XMLSignature signature = factory.newXMLSignature(si, ki);
 
     signature.sign(dsc);
-    return marshallToString(document);
```

Comment:
Refactored this method so that if the assertion is signed and then encrypted, we don't have to marshall and unmarshall and re-marshall.